applications/starlay_finance.md

```diff
@@ -0,0 +1,143 @@
+
+Seiya Chida (COO)
+
+David Lee (Lead Dev)
+
+And three other devs
+
+
+### Contact
+
+- **Contact Name:** Seiya Chida
+- **Contact Email:** seiya@starlay.finance
+- **Website:** https://starlay.finance/
+
+### Team's experience
+
+We are Starlay Finance, a team of experienced DeFi developers, designers and marketers. Starlay Finance once ranked as a huge DeFi locked with more than $500M in just 5 weeks after its launch on Astar Network. We've also developed Kagla(Stable swap) and Muuu(Yield booster) on Astar Network. 
+
+### Team Code Repos
+
+- https://github.com/starlay-finance/<Starlay Finance>
+- https://github.com/kagla-finance/<Kagla Finance>
+- https://github.com/muuu-finance/<Muuu Finance>
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- https://github.com/artat9
+- https://github.com/un-ast
+- https://github.com/wolfwarrier14
+- https://github.com/qudacki
+- https://github.com/chidasan
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/shumpeikoike/
+- https://www.linkedin.com/in/seiya-chida-7a9717165/
+
+## Development Status :open_book:
+
+We have completed the basic implementation and are currently adding test code and making minor modifications. As Grant progresses, we plan to request an audit firm to conduct a code audit.
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 12 weeks
+- **Full-Time Equivalent (FTE):**  2FTE, 4PTE
+- **Total Costs:** 180,000 - 230,000 USD
+
+### Milestone 1 — Basic functionality
+
+- **Estimated duration:** 5 weeks
+- **FTE:**  2FTE, 4PTE
+- **Costs:** 50,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide both documentation of the code and a basic video tutorial that explains how a user can easily use Starlay app |
+| 1. | Deposit function| Allow users to deposit assets into our lending platform and receive interest-bearing tokens |
+| 2. | Borrow function| Allow users to borrow an asset at a low-interest rate by offering the users assets as collateral |
+| 2. | Mint function| Allow users to supply assets to a Starlay market and earn interest on their deposits|
+| 3. | Redeem function| Allow users to convert their lTokens (lay ibTokens) back into the underlying assets they initially supplied |
+| 4. | Repay_borrow function| Allow users to repay their loans and release the collateral that was locked during the borrowing process |
+| 5. | Liquidate_borrow function |　Repay a part of the borrower's outstanding loan on behalf of the borrower　|
+| 6. | List tokens (by the onwer)| Allow users to deposit and borrow ERC-20 based tokens as well as the native token(ASTR) |
+| 7. | Switch between EVM and Polkadot| Allow users to switch between EVM and Polkadot wallet |
+
+
+
+### Milestone 2 - Audit
```

Comment:
FYI, this would be something that the ink!ubator would potentially organize and pay for directly. I.e we would not pay you to then pay an auditor

Reply:
Great! We are gonna ask Hacken to do our first round of code audits. I will share the estimated cost that I just received from them. If you need a more detailed conversation, please feel free to contact me anytime through Telegram (https://t.me/seiya_starlay), or  email(seiya@starlay.finance).

Reply:
@Chidasan 
As I am aware, Hacken team will apply for the Auditor track, and then we could assign them to do an audit, thank you